build.sc

```diff
@@ -0,0 +1,55 @@
+import $ivy.`com.lihaoyi::mill-contrib-scoverage:`
+import mill.contrib.scoverage.ScoverageModule
+import $ivy.`com.goyeau::mill-git::0.2.5`
+import com.goyeau.mill.git.GitVersionedPublishModule
+import mill._
+import mill.scalalib._
+import mill.scalalib.publish._
+
+trait SopeModule extends GitVersionedPublishModule with SbtModule with ScoverageModule {
+  override def scalaVersion = "2.12.18"
+  def sparkVersion = "3.1.3"
+  def scalaTestVersion = "3.2.16"
+  def jacksonYamlVersion = "2.6.5"
+  def bigqueryConnectorVersion = "hadoop3-0.13.11"
+  def scoptVersion = "3.7.1"
+  def scoverageVersion = "2.0.10"
+
+  def pomSettings = PomSettings(
+    organization = "com.mayurb",
+    description = "Sope",
+    url = "https://github.com/mayurbhangale/sope",
```

Comment:
My github id is incorrect, will fix it.

Reply:
I fixed it. Not sure where I got the wrong values from...